src/blocks/tipset.rs

```diff
@@ -142,8 +143,8 @@ impl Tipset {
     pub fn load(store: impl Blockstore, tsk: &TipsetKeys) -> anyhow::Result<Option<Tipset>> {
         Ok(tsk
```

Comment:
How about directly iterating over `tsk`?

Reply:
I believe the recent changes address this, but let me know if you were thinking of something else.